Sources/ClientRuntime/Middleware/RetryMiddleware.swift

```diff
@@ -45,7 +64,24 @@ public struct RetryMiddleware<Strategy: RetryStrategy,
                 // TODO: log token error here
                 throw operationError
             }
-            return try await sendRequest(token: token, context: context, input: input, next: next)
+            guard let estimatedSkew: TimeInterval = context.attributes.get(key: AttributeKeys.estimatedSkew) else {
+                throw ClientError.dataNotFound("Estimated skew not found; failed to calculate TTL for retry.")
```

Comment:
Do we want to fail the entire request for lack of a parameter for this header?
Not sure what could cause this, but I don't think we want to fail customer requests for lack of this header.

Reply:
Changed to logging .info level message with logger & using default values.